### DISCUSSION TITLE
REI not showing

### DISCUSSION THREAD

**Discussion by NotAthxna** [shedaniel/RoughlyEnoughItems#925] [Q & A]
Hi since I installed the 1.19 version of REI it doesnt seems to show up when I press inventory, I can't find the related keybinds and I can't configurate it through ModMenu. Maybe I toggled it off on accident? Is there any solution like the original key to toggle ?

**Comment by shedaniel** [comment]
Does the recipe book show up with any recipes? REI usually doesn't show up if the recipe book also died.
